pkg/taxonomy/document.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package taxonomy
+
+import (
+	"path/filepath"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+// ValidateDocument validates input json data against a taxonomy file
+func ValidateDocument(taxonomyFile string, jsonData string) (*gojsonschema.Result, error) {
```

Comment:
Is this meant to be used only by the tool that validates taxonomies via the cli?  If not, I imagine runtime validation would not be done from a file.  Right?